akd/Cargo.toml

```diff
@@ -62,7 +62,6 @@ once_cell = { version = "1" }
 ctor = "0.1"
 tokio-test = "0.4"
 tokio = { version = "1.21", features = ["rt", "sync", "time", "macros"] }
-akd = { path = ".", features = ["public-tests"], version = "0.8.3", default-features = false }
```

Comment:
@slawlor Is this fine to remove?

Reply:
No just remove the version piece of it. In a test profile, we want to reference the latest code we have so we don't specify a version number and therefore it won't try to ever pull from crates.io but will build from local source code

Reply:
We don't want to include the public tests feature always, hence why we re-import the library under the Dev profile saying the default in Dev, also known as test, is to enable the public tests feature. This way you can build, and have a default set of features, which is different from the test profile

Reply:
Ah that makes sense! The reason why we have this line is also manifesting in CI. I wonder why we included a version number in the first place...

Reply:
Iirc we never had a version number did we? Anyways this should fix it

Reply:
Looks like we didn't have a version number for a while, then 0.8.0 was added in #292. It might have been an accident, let's revert it to version-less!